CONTRIBUTING.md

```diff
@@ -14,15 +14,44 @@ purposes. As such there are repository "maintainers" who are responsible for
 merging pull requests as well as a "lead maintainer" who is responsible for the
 release cycle, overall merging, moderation and appointment of maintainers.
 
+Getting Started
+---------------
+
+New developers are very welcome and needed. There are a lot of open issues of
+any difficulty waiting to be fixed. However, before you start contributing,
+familiarize yourself with the Bitcoin Core build system and tests. Refer to the
+documentation in the repository on how to build Bitcoin Core and how to run the
+unit and functional tests.
+
 If you're looking for somewhere to start contributing, check out the
 [good first issue](https://github.com/bitcoin/bitcoin/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
-list or participate in a weekly [Bitcoin Core PR Review Club](https://bitcoincore.reviews/) meeting.
+list or changes that are
+[up for grabs](https://github.com/bitcoin/bitcoin/issues?utf8=%E2%9C%93&q=label%3A%22Up+for+grabs%22).
+Some of them might no longer be applicable. So if you are interested, but
+unsure, you might want to leave a comment on the issue first.
+
+You may also participate in the weekly
+[Bitcoin Core PR Review Club](https://bitcoincore.reviews/) meeting.
+
+Finally, review and testing is important as well. It gives you an idea of what
+others are working on, where you can help, and how the review process works.
+Experience from testing and reviewing will help you become a better
+contributor. Please refer to the section [peer review](#peer-review) later in
+this document.
+
+### Good First Issue Label
+
+The purpose of the `good first issue` label is to highlight which issues are suitable for a new contributor without a deep understanding of the codebase.
+
+However, `good first issue`s can be solved by anyone. If they remain unsolved for a longer time, a common contributor might address them.
+
+You do not need to request permission to start working on this. You are encouraged to comment on the issue if you are planning to work on it. This will help other contributors monitor which issues are actively being addressed and is also an effective way to request assistance if and when you need it.
```

Comment:
```suggestion
You do not need to request permission to start working on an issue. However, you are encouraged to leave a comment if you are planning to work on it. This will help other contributors monitor which issues are actively being addressed and is also an effective way to request assistance if and when you need it.
```